<ui version="4.0" >
 <class>AuthorizationDialogClass</class>
 <widget class="QDialog" name="AuthorizationDialogClass" >
  <property name="windowModality" >
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>294</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="styledFrame" >
     <property name="frameShape" >
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth" >
      <number>0</number>
     </property>
     <layout class="QVBoxLayout" name="frameVLayout" >
      <property name="spacing" >
       <number>5</number>
      </property>
      <property name="leftMargin" >
       <number>1</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>1</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" >
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>4</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_4" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>10</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="closeToolButton" >
          <property name="focusPolicy" >
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="toolTip" >
           <string>CloseButtonHint</string>
          </property>
          <property name="accessibleName" >
           <string>transparentButton</string>
          </property>
          <property name="layoutDirection" >
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="popupMode" >
           <enum>QToolButton::DelayedPopup</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="mainDialogFrame" >
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Expanding" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth" >
         <number>0</number>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <item>
          <layout class="QGridLayout" name="gridLayout" >
           <property name="horizontalSpacing" >
            <number>12</number>
           </property>
           <property name="verticalSpacing" >
            <number>0</number>
           </property>
           <property name="margin" >
            <number>9</number>
           </property>
           <item row="0" column="0" colspan="2" >
            <widget class="QLabel" name="topLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font" >
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>9</pointsize>
               <weight>50</weight>
               <italic>false</italic>
               <bold>false</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text" >
              <string>TopText_DT</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <spacer name="verticalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType" >
              <enum>QSizePolicy::Fixed</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>20</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="2" column="1" >
            <widget class="QLabel" name="authUserNameLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font" >
              <font>
               <family>MS Shell Dlg 2</family>
               <pointsize>11</pointsize>
               <weight>75</weight>
               <italic>false</italic>
               <bold>true</bold>
               <underline>false</underline>
               <strikeout>false</strikeout>
              </font>
             </property>
             <property name="text" >
              <string>UserName_DT</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="3" column="0" >
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <widget class="MoodBox::ContactAvatarButton" name="contactAvatarToolButton" >
               <property name="focusPolicy" >
                <enum>Qt::NoFocus</enum>
               </property>
               <property name="accessibleName" >
                <string>transparentButton</string>
               </property>
               <property name="iconSize" >
                <size>
                 <width>74</width>
                 <height>74</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="3" column="1" >
            <widget class="QPlainTextEdit" name="messageEdit" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize" >
              <size>
               <width>16777215</width>
               <height>70</height>
              </size>
             </property>
             <property name="frameShape" >
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow" >
              <enum>QFrame::Plain</enum>
             </property>
             <property name="verticalScrollBarPolicy" >
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy" >
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer" >
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>0</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="topMargin" >
         <number>9</number>
        </property>
        <property name="rightMargin" >
         <number>9</number>
        </property>
        <property name="bottomMargin" >
         <number>9</number>
        </property>
        <item>
         <spacer name="horizontalSpacer" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="okButton" >
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>Send_DT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="rejectButton" >
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>Reject_DT</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelButton" >
          <property name="accessibleName" >
           <string>rectButton</string>
          </property>
          <property name="text" >
           <string>Cancel_DT</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>MoodBox::ContactAvatarButton</class>
   <extends>QToolButton</extends>
   <header>contactavatarbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>messageEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>rejectButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
